tests/app/dao/test_services_dao.py

```diff
@@ -625,6 +625,31 @@ def test_fetch_stats_for_today_only_includes_today(notify_db, notify_db_session,
     assert stats['created'] == 1
 
 
+def test_fetch_stats_should_not_gather_notifications_older_than_7_days(notify_db, notify_db_session, sample_template):
+    with freeze_time('2001-01-01T12:00:00'):
+        # 8 days ago
+        create_notification(notify_db, None, to_field='1', status='delivered')
```

Comment:
If you use the function that is aliased create_notification_db you can pass in a created_at date and not need to use freeze_time